cmd/cli/plugin/package/package_available_get.go

```diff
@@ -113,20 +115,20 @@ func packageAvailableGet(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
-func getValuesSchema(cmd *cobra.Command, args []string, kc kappclient.Client) error {
-	if pkgVersion == "" {
+func getValuesSchemaForPackage(namespace, name, version string, kc kappclient.Client, writer io.Writer) error {
+	if version == "" {
 		return errors.New("version is required when --values-schema flag is declared. Please specify <PACKAGE-NAME>/<VERSION>")
 	}
-	pkg, pkgGetErr := kc.GetPackage(fmt.Sprintf("%s.%s", pkgName, pkgVersion), packageAvailableOp.Namespace)
+	pkg, pkgGetErr := kc.GetPackage(fmt.Sprintf("%s.%s", name, version), namespace)
 	if pkgGetErr != nil {
 		if apierrors.IsNotFound(pkgGetErr) {
-			return errors.Errorf("package '%s/%s' does not exist in the '%s' namespace", pkgName, pkgVersion, packageAvailableOp.Namespace)
+			return errors.Errorf("package '%s/%s' does not exist in the '%s' namespace", name, version, namespace)
```

Comment:
Using global variable `packageAvailableOp.Namespace` makes it is hard for unit tests. So replacing with local variable.